documentation/staging/content/userguide/managing-domains/model-in-image/common-mounts.md

```diff
@@ -0,0 +1,678 @@
+
+This section describes a typical common mount configuration for the
+Model in Image use case.
+
+#### Common mount images
+
+One or more common mount images can be configured on a domain resource `serverPod`.
+A `serverPod` can be defined at the domain scope, which applies to every pod in
+the domain, plus the introspector job's pod, at a specific WebLogic cluster's scope,
+or at a specific WebLogic Server pod's scope. Typically, the domain scope is
+the most applicable for the Model in Image use case; for example:
+
+```
+spec:
+  serverPod:
+    commonMounts:
+    - image: model-in-image:v1
+      imagePullPolicy: IfNotPresent
+      volume: commonMountsVolume1
+```
+
+{{% notice note %}}
+If image pull secrets are required for pulling common mounts images,
+then the secrets must be referenced using `domain.spec.imagePullSecrets`.
+{{% /notice %}}
+
+#### Common mount volumes and paths
+
+The `serverPod.commonMounts.volume` field refers to the name of a common
+mount volume defined in the `domain.spec.commonMountVolumes` section, and
+a common mount volume, in turn, defines a `mountPath`. The `mountPath`
+is the location of a directory in a common mount image, and
+is also the location in the main pod container (which will automatically contain
+a recursive copy of the common mount image directory). For example:
+
+```
+  spec:
+    commonMountVolumes:
+    - name: commonMountsVolume1
+      mountPath: /common
+```
+
+#### Model in Image paths
+
+For the Model In Image common mount use case, you also need to
+configure the `domain.spec.configuration.model.modelHome`
+and `domain.spec.configuration.model.wdtInstallHome` attributes
+to specify the location of the domain's WebLogic Deploy Tool (WDT)
+model files and the domain's WDT installation.
+These default to `/u01/wdt/models` and `/u01/wdt/weblogic-deploy`
+respectively, and must be changed to specify a directory in
+`domain.spec.commonMountVolumes.mountPath`. For example:
+
+```
+  configuration:
+    model:
+      modelHome: "/common/models"
+      wdtInstallHome: "/common/weblogic-deploy"
+```
+
+### Sample
+
+This sample demonstrates deploying a Model in Image domain that uses
+common mounts to supply the domain's WDT model files,
+application archive ZIP files, and WDT installation in a small, separate
+container image.
+
+#### Step 1: Prerequisites
+
+- First, follow all of the steps in the Model in Image
+  [initial use case sample](/weblogic-kubernetes-operator/samples/simple/domains/model-in-image/initial/).
+
+  This will:
+
+  - Set up the operator and a namespace for the domain.
+  - Download a WebLogic Deploy Tool ZIP installation.
+  - Deploy a domain _without_ common mounts.
+
+- Second, shut down the domain and wait for its pods to exit.
+  - You can use the `wl-pod-wait.sh` script to wait.
+  - For example, assuming that
+    you have set up `/tmp/mii-sample` as your working directory:
+    ```shell
+    kubectl delete domain sample-domain1 -n sample-domain1-ns
```

Comment:
Add $ at beginning of shell command line

Reply:
Fixed.